lib/js/dom.nim

```diff
@@ -1235,6 +1237,11 @@ proc slice*(e: Blob, startindex: int = 0, endindex: int = e.size, contentType: c
 # Performance "methods"
 proc now*(p: Performance): float
 
+# Selection "methods"
+proc removeAllRanges*(s: Selection)
+converter toString*(s: Selection): cstring
+proc `$`*(s: Selection): string = $(s.toString())
```

Comment:
this causes RT error `Uncaught TypeError: a_2990031.$ is not a function` 
https://www.dropbox.com/s/fae2gusepi51drt/Screenshot%202020-03-31%2019.18.16.png?dl=0

see test case https://github.com/timotheecour/vitanim/blob/master/testcases/tests/t10467/readme.md

the bug is that it's inside a importcpp section so gets interpreted as such